src/main/java/com/iridium/iridiumskyblock/generators/OceanGenerator.java

```diff
@@ -87,15 +88,15 @@ public void generateWater(World world, int x, int z) {
         int currentFloorHeight = (int) ((generator.noise(x, z, 1.5D, 0.5D, true) + 1) * (maxOceanFloorLevel - minOceanFloorLevel) + minOceanFloorLevel);
 
         // Generate layer of bedrock
-        if (world.getBlockAt(x, 0, z).getType() != XMaterial.BEDROCK.parseMaterial()) {
-            if (world.getBlockAt(x, 0, z).getState() instanceof InventoryHolder) {
-                ((InventoryHolder) world.getBlockAt(x, 0, z).getState()).getInventory().clear();
+        if (world.getBlockAt(x, LocationUtils.getMinHeight(world), z).getType() != XMaterial.BEDROCK.parseMaterial()) {
+            if (world.getBlockAt(x, LocationUtils.getMinHeight(world), z).getState() instanceof InventoryHolder) {
+                ((InventoryHolder) world.getBlockAt(x, LocationUtils.getMinHeight(world), z).getState()).getInventory().clear();
             }
-            world.getBlockAt(x, 0, z).setType(Material.BEDROCK, false);
+            world.getBlockAt(x, LocationUtils.getMinHeight(world), z).setType(Material.BEDROCK, false);
         }
 
         // Generate gravel layer
-        for (int y = 1; y < currentFloorHeight; y++) {
+        for (int y = LocationUtils.getMinHeight(world) + 1; y < currentFloorHeight; y++) {
```

Comment:
You use this method that often here... Just introduce a variable

Reply:
resolved